src/helpers/constants.py

```diff
@@ -0,0 +1,9 @@
+QUEST_COLUMN_NAME = "Quests"
+EXPERIENCE_COLUMN_NAME = "XP"
+BOX_WRAPPER_CHAR = "="
+BOX_SIDE_CHAR = "||"
+PRE_TEXT = BOX_SIDE_CHAR + "  "
+POST_TEXT = "  " + BOX_SIDE_CHAR
```

Comment:
Rename to include that these are used in block text
`BOX_PRE_TEXT` and `BOX_POST_TEXT`

Reply:
updated